public/app/features/explore/hooks/useStateSync/migrators/v0.ts

```diff
@@ -12,13 +11,52 @@ export interface ExploreURLV0 extends BaseExploreURL {
 
 export const v0Migrator: MigrationHandler<never, ExploreURLV0> = {
   parse: (params) => {
-    return {
-      schemaVersion: 0,
-      left: parseUrlState(typeof params.left === 'string' ? params.left : undefined),
-      ...(params.right && {
-        right: parseUrlState(typeof params.right === 'string' ? params.right : undefined),
-      }),
-    };
+    // If no params are provided, return the default state without errors
+    // This means the user accessed the explore page without any params
+    if (!params.left && !params.right) {
+      return [
+        {
+          left: {
+            datasource: null,
+            queries: [],
+            range: {
+              from: 'now-6h',
+              to: 'now',
+            },
+          },
+          schemaVersion: 0,
+        },
+        false,
+      ];
+    }
+
+    let left: ExploreUrlState | undefined;
+    let right: ExploreUrlState | undefined;
+    let leftError, rightError: boolean | undefined;
+
+    if (typeof params.left === 'string') {
+      [left, leftError] = parsePaneState(params.left);
+    }
+
+    if (typeof params.right === 'string') {
+      [right, rightError] = parsePaneState(params.right);
+    } else if (params.right) {
+      right = FALLBACK_PANE_VALUE;
+      rightError = true;
+    }
+
+    if (!left) {
+      left = FALLBACK_PANE_VALUE;
+    }
+
+    return [
+      {
+        schemaVersion: 0,
+        left,
+        ...(right && { right }),
+      },
+      leftError || !!rightError,
```

Comment:
Why not:

```suggestion
      !!leftError || !!rightError,
```

Reply:
changed it for clarity but it's not really needed
`undefined || something` always evaluates to `something`, that's why i doubled banged only `rightError`